api/src/main/resources/public/swagger/api.yaml

```diff
@@ -1384,6 +1407,9 @@ components:
           type: number
         dueDate:
           type: string
+        paid:
```

Comment:
One thing that could be done better here would be to add an `allOf` reference to a `Payment` schema for both the `MemberPayment` and `AdminPayment` schema here. Since there was already a `Payment` entity class in the code, I wasn't really sure what to name the new schema, so I decided to just add the new field in both object.